bindings/netstandard/ElectionGuard/ElectionGuard.Decryption.Tests/Decryption/DecryptionMediatorExtensions.cs

```diff
@@ -0,0 +1,58 @@
+using ElectionGuard.Decryption.Decryption;
+using ElectionGuard.Decryption.Tests.Tally;
+using ElectionGuard.ElectionSetup;
+using ElectionGuard.Decryption.ChallengeResponse;
+using ElectionGuard.Decryption.Shares;
+
+namespace ElectionGuard.Decryption.Tests.Decryption;
+
+public static class DecryptionMediatorExtensions
+{
+    /// <summary>
+    /// Run the decryption process
+    /// </summary>
+    public static DecryptionResult RunDecryptionProcess(
+    this DecryptionMediator mediator, TestDecryptionData data, List<Guardian> quorumOfGuardians)
```

Comment:
this serves as an example of running the entire process with the mediator. it can be used as the primary example of how the api's work and what they expect.